pkg/util/path.go

```diff
@@ -130,6 +131,21 @@ func RemoveBrackets(pe string) (string, bool) {
 	return pe[1 : len(pe)-1], true
 }
 
+// ParseValue parses string into a value
```

Comment:
nit: this function more suitable to be in https://github.com/istio/operator/blob/master/pkg/util/util.go

Reply:
Sure, will move it.

Reply:
Updated.